t/t-multiple-remotes.sh

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env bash
+
+. "$(dirname "$0")/testlib.sh"
+
+reponame="$(basename "$0" ".sh")"
+
+prepare_mrtest(){
+  MRHOME=$TRASHDIR/home/mrhome
+  rm -rf $MRHOME
+  mkdir $MRHOME
+  HOME=$MRHOME
+  git config --global user.name "Git LFS Tests"
+  git config --global user.email "git-lfs@example.com"
+}
+
+prepare_consumer() {
+  local consumer=$1
+  mkdir $consumer
+  cd $consumer
+  git init
+  git remote add mr file://$smain
+  git remote add fr file://$sfork
+  git fetch mr
+  git fetch fr
+  git lfs install
+}
+
+prepare_remote() {
+  local repodir="$1"
+  mkdir -p "$repodir"
+  cd "$repodir"
+  git init --bare
+  git config http.receivepack false
+  git lfs install
+}
+
+prepare_forks () {
+  smain=$HOME/$reponame"-main-remote.git"
+  sfork=$HOME/$reponame"-fork-remote.git"
+  cmain=$HOME/$reponame"-main-repo"
+  cfork=$HOME/$reponame"-fork-repo"
+  rm -rf $smain
+  rm -rf $sfork
+  rm -rf $cmain
+  rm -rf $cfork
```

Comment:
Instead of deleting and recreating repos like this, let's use a parameter which is specific to each call and set that differently in each individual test.  Then we can avoid different tests affecting each other and let the cleanup code at the end of the test clean up all of the test data for us.

You can see an example in `t/t-pull.sh` how we do this with `setup_remote_repo`.

Reply:
I agree. That looks cleaner. I changed the new function prepare_forks to create remote and local repos with different paths
`prepare_forks test_case`